magento/magento2
# Customer access token API triggers same events as regular customer frontend login

### Description (*)

Now the `Magento\Integration\Model\CustomerTokenService::createCustomerAccessToken()` will dispatch the same events as the regular customer frontend login so possible customer observers without checking `instanceof` may work correctly. 
- Event `customer_data_object_login` contains the customer data model `Magento\Customer\Api\Data\CustomerInterface`
- Event `customer_model` now contains the customer model `Magento\Customer\Model\Customer`

### Related Pull Requests


### Fixed Issues (if relevant)
1. Fixes magento/magento2#29456

### Manual testing scenarios (*)
1. Create customer account via storefront or REST API
2. Execute POST {{url}}/rest/V1/integration/customer/token request with your username and password
3. customer_login and customer_data_object_login events should be triggered with correct customer models.

### Questions or comments


### Contribution checklist (*)
